singleton-service/src/service/auth.rs

```diff
@@ -72,8 +77,9 @@ pub fn build_auth_request(auth: &Auth) -> Result<Request, anyhow::Error> {
         }
     }
     let txn = vec![(Transaction::new(&app, None, None, None))];
-    // TODO : Enable list keys extension.
-    let extensions = extensions::List::new().push(extensions::Extension::Hierarchy);
+    let extensions = extensions::List::new()
+        .push(extensions::Extension::Hierarchy)
+        .push(extensions::Extension::ListAppKeys(1.to_string().into()));
```

Comment:
You should be able to use `"1".into()`, since `&'static str` can become a `Cow<'static, str>` (and reference static memory rather than allocate a new string).

Reply:
fixed for all the places